### PR TITLE
Refactor story block

### PR DESCRIPTION
**Pull request by easherma**
Simplying story block component, then realized we need to make sure it stacks on mobile devices

### CODE REVIEW

File: src/components/story-block.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const StoryBlock = ({
+  imageSource,
+  altText,
+  description,
+  imageRight = false,
+}) => (
+  <section>
+    <Container>
+      <Row>
+        <Col sm={{ span: '6', order: `${imageRight ? 'last' : 'first'} ` }}>
```
Comment (easherma) [inline src/components/story-block.js:17]: Checkout the 'order' param here: https://react-bootstrap.github.io/layout/grid/#responsive-grids

And here we're using a nice friendly version of the ternary operator. The last shall be first and the first shall be last. Very Biblical.

File: src/components/story-block.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <Img fluid={imageSource} alt={altText} />
```
Comment (easherma) [inline src/components/story-block.js:18]: using the new gatsby image component, called 'Img'

File: src/pages/index.js [hunk 3]
```diff
@@ -30,6 +31,13 @@ const IndexPage = () => {
           }
         }
       }
+      barriers: file(relativePath: { eq: "images/barriers.png" }) {
```
Comment (easherma) [inline src/pages/index.js:34]: when we sync up we can talk about this whole thing

File: src/pages/index.js [hunk 4]
```diff
@@ -44,6 +52,17 @@ const IndexPage = () => {
...
+      <StoryBlock
+        imageSource={images.barriers.childImageSharp.fluid}
+        altText="Barriers"
+        description="Hellow word!"
+      />
+      <StoryBlock
+        imageSource={images.barriers.childImageSharp.fluid}
+        altText="Barriers"
+        description="Hellow word!"
+        imageRight
```
Comment (easherma) [inline src/pages/index.js:64]: see here, since the prop is a boolean value (true or false), but just including it we mean 'true' (imageRight=true)

**Review by easherma — COMMENTED** [review]
check it out

File: src/components/story-block.js (hunk 1 above)
Comment (ToniTepes) [reply src/components/story-block.js:17]: Ok. Is the order param kind of like a media query?

And the ternary operator in this case is the `imageRight` ?

File: src/components/story-block.js (hunk 2 above)
Comment (ToniTepes) [reply src/components/story-block.js:18]: Helps control the image?

**Review by ToniTepes — COMMENTED** [review]
Commented where I would have question, for my own reference.

File: src/components/story-block.js (hunk 1 above)
Comment (easherma) [reply src/components/story-block.js:17]: Great question! That's basically correct:
order
The React JSX code compiles to html, which adds these bootstrap class names, that link to their stylesheet where they've handled most of the media query stuff. 

For the ternary operator: 

```imageRight ? 'last' : 'first'```

That is it. 
The brackets around it with the dollar sign make a template literal: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

So its basically javascript ternary operator inside a javascript expression that is a template literally inside a javascript expression that is the whole thing in double brackets.

File: src/components/story-block.js (hunk 2 above)
Comment (easherma) [reply src/components/story-block.js:18]: Img does all the fancy optimization stuff: 
https://www.gatsbyjs.org/packages/gatsby-image/

**Pull request merged**